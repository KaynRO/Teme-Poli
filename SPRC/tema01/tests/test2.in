login kayn
load
del 2
del 5
get_stat 1
get_stat 2
get_stat_all
del 1
del 3
store
logout
