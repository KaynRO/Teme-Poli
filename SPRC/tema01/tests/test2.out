--------------------------------
User already authenticated
--------------------------------
Login in first
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
Logout successfully
